### PR TITLE
Television mock-up created

### PR DESCRIPTION
**Pull request by gauravJoshi1992**

### CODE REVIEW

File: src/reducers/television.reducers.js [hunk 1]
```diff
@@ -0,0 +1,1625 @@
...
+				"USB ports": 2,
+				"HDMI ports": 2,
+				"internet": true,
+			},
+			"HD": "Full HD",
+			"weight": 4.6,
+			"color": "Black",
+			"model": "Samsung32J5500AK",
+			"brand": "Samsung",
+			"Power Consmption": 90,
```
Comment (priyanka271990) [inline src/reducers/television.reducers.js:26]: Power Consumption

Comment (gauravJoshi1992) [reply src/reducers/television.reducers.js:26]: @priyanka271990 Its always in watt, you need to static string of "W" into it.

Comment (priyanka271990) [reply src/reducers/television.reducers.js:26]: ok. But the spelling of consumption is wrong.

Comment (gauravJoshi1992) [reply src/reducers/television.reducers.js:26]: Oh good catch. Just correct it and create the pr.

**Pull request merged**